portal/src/graphql/portal/PasswordsScreen.tsx

```diff
@@ -36,6 +42,61 @@ const PasswordsScreen: React.FC = function PasswordsScreen() {
         };
   }, [data]);
 
+  const [currentPivotKey, setCurrentPivotKey] = useState<PivotKey>(PASSWORD_POLICY_PIVOT_KEY);
+  const selectedPivotKeyRef = useRef<PivotKey>(PASSWORD_POLICY_PIVOT_KEY);
```

Comment:
We are treating the Pivot as a controlled component. Why do we need to keep ref? If we do not setCurrentPivotKey in onLinkClick, the pivot should not change at all, correct? The logic here is a bit long and complicated.

Reply:
@louischan-oursky We need to switch to other tab when user click confirm button. So need to store the selected key.